rollup.config.js

```diff
@@ -0,0 +1,68 @@
+import svelte from "rollup-plugin-svelte";
+import resolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import { terser } from "rollup-plugin-terser";
+import sveltePreprocess from "svelte-preprocess";
+import typescript from "@rollup/plugin-typescript";
+import path from "path";
+import fs from "fs";
+
+const production = !process.env.ROLLUP_WATCH;
```

Comment:
Where is this env variable defined?

Reply:
It's not defined as of now. I think it comes to play when we are in production. But I have to look more into it.

Reply:
It's defined by Svelte, we don't need to worry about this